protocol/x/perpetuals/lib/lib_test.go

```diff
@@ -421,36 +421,16 @@ func TestGetMarginRequirementsInQuoteQuantums_2(t *testing.T) {
 		bigExpectedInitialMargin     *big.Int
 		bigExpectedMaintenanceMargin *big.Int
 	}{
-		"InitialMargin 2 BIPs, MaintenanceMargin 1 BIP, positive exponent, atomic resolution 8": {
```

Comment:
nit: Add TODO for revisiting adding back positive exponents once supported by x/marketmap?

Reply:
done